news/8848.doc

```diff
@@ -0,0 +1 @@
+8783.doc
```

Comment:
N.B. It's a symlink to the other fragment.

Reply:
Would it be worthwhile to we have the same text copied over instead? That might just be less fragile if someone's on Windows or whatever. :3

Reply:
Does anybody on windows generate changelogs? I'd address this only if such a problem actually occurs...

Reply:
Yes, I would do so if I were the one making a release. IMO, we shouldn't have symlinks in the pip repository. If that's not already a project standard, I'd be inclined to make it one. Maintainers do their best to make sure the release process works on Windows, but there's always the odd glitch that I need to fix - so anything we can do to reduce the risk of such things happening, we should IMO.

Is there a pre-commit hook we can add to check for and reject symlinks in the repo?

Reply:
I only saw a hook that checks that symlinks point to things that actually exist...

P.S. Just out of curiosity, could you please check if the symlink actually breaks anything on Windows? (I don't think it does but still, it'd be good to know)

Reply:
Oh, and the reason I'm using a symlink is just that if somebody wants to update the message, it'd be updated in both places, not just one.

Reply:
> Just out of curiosity, could you please check if the symlink actually breaks anything on Windows? 

Not really right at the moment, I'm not set up to try running the release scripts right now. I'll see if I can set something up at some point.

BTW, how does git even do symlinks if you check out on a filesystem that doesn't support them?

Reply:
Looks like it just turns them into text files
https://stackoverflow.com/questions/5917249/git-symlinks-in-windows